### PR TITLE
Always output TOTAL line.

### PR DESCRIPTION
**Pull request by lordjabez**
Always output the TOTAL line, and update tests to match new functionality.

### CODE REVIEW

File: coverage/summary.py [hunk 1]
```diff
@@ -120,8 +120,8 @@ def report(self, morfs, outfile=None):
         for line in lines:
             self.writeout(line[0])
 
-        # Write a TOTAl line if we had more than one file.
-        if self.total.n_files > 1:
+        # Write a TOTAl line if we had at least one file.
```
Comment (portante) [inline coverage/summary.py]: s/TOTAl/TOTAL/

**Review by portante — APPROVED** [review]
FWIW ...

File: coverage/summary.py (hunk 1 above)
Comment (lordjabez) [reply coverage/summary.py]: Ugh, I literally made that mistake a second time didn't i?

**Review by portante — APPROVED** [review]

File: CHANGES.rst [hunk 2]
```diff
@@ -24,7 +24,7 @@ want to know what's different in 5.0 since 4.5.x, see :ref:`whatsnew5x`.
 Unreleased
 ----------
 
-Nothing yet.
+- Always output TOTAL line.
```
Comment (nedbat) [inline CHANGES.rst]:
```suggestion
- The text report produced by ``coverage report`` now always outputs a TOTAL line, even if only one Python file is reported.  This makes regex parsing of the output easier.  Thanks, Judson Neer.
```

File: tests/test_process.py [hunk 3]
```diff
@@ -1348,7 +1352,9 @@ def test_accented_directory(self):
         report_expected = (
             u"Name            Stmts   Miss  Cover\n"
             u"-----------------------------------\n"
-            u"\xe2%saccented.py       1      0   100%%\n" % os.sep
+            u"\xe2%saccented.py       1      0   100%%\n"
+            u"-----------------------------------\n"
+            u"TOTAL               1      0   100%%\n" % os.sep
```
Comment (nedbat) [inline tests/test_process.py]: You don't need the string formatting and os.sep on this line.

**Comment by nedbat** [comment]
This is great, thanks.  A little more detail in the changelog, and remove an unneeded string formatting, and we are ready to merge!

File: tests/test_process.py (hunk 3 above)
Comment (lordjabez) [reply tests/test_process.py]: If I don't move the `% os.sep` piece to the last line (or after the trailing parenthesis) the code fails to execute with a syntax error. And the substitution has to be _somewhere_

I'll tweak this a bit and hopefully it's more obvious what's going on.

**Comment by nedbat** [comment]
@lordjabez Thanks. I see now that I misread the os.sep string formatting, thanks for putting up with me :)  I will merge this when the tests pass.

**Pull request merged**